pkg/nseconfig/config.go

```diff
@@ -37,6 +39,10 @@ type VL3 struct {
 	DNSZones    []string `yaml:"dnsZones"`
 }
 
+type PassThrough struct {
+	Ifname	string	`yaml:"ifName"`
```

Comment:
It might be better to name this field `IfName` (and change the one in `VL3` as well) to make the field names consistent - all other fields follow _camelCase_/_CamelCase_ with `yaml` struct tags, but this one.